src/main.rs

```diff
@@ -177,8 +177,13 @@ fn main() -> crossterm::Result<()> {
     }
 
     if opt.list_languages {
-        opt.languages()
-            .expect("Couldn't get installed languages under config directory. Make sure the config directory exists.")
+        match opt.languages() {
```

Comment:
Since `or_else` methods on results don't  return a default, an `if` or a `match` is necessary

Reply:
_(improving the explanation)_ the `or_else` will return another Result which has to be handled.